azure-devops/projects/pagopa-projects/pagopa-api-config.tf

```diff
@@ -98,5 +116,32 @@ module "pagopa-api-config_deploy" {
 
   service_connection_ids_authorization = [
     azuredevops_serviceendpoint_github.io-azure-devops-github-ro.id,
+    azuredevops_serviceendpoint_azurerm.DEV-PAGOPA.id,
+    azuredevops_serviceendpoint_azurerm.UAT-PAGOPA.id,
+  ]
+}
+
+
+module "pagopa-api-config_newman-test" {
+  source = "git::https://github.com/pagopa/azuredevops-tf-modules.git//azuredevops_build_definition_code_review?ref=v0.0.3"
```

Comment:
if you use [this template](https://github.com/pagopa/azuredevops-tf-modules/blob/e991d898ce9130cfc2cf2dee999d2ec8d28fec7d/azuredevops_build_definition_code_review/main.tf#L26) you can't define your project pipeline `devops/newman-test-pipeline.yml` see [PR](https://github.com/pagopa/pagopa-api-config/pull/15).
I think could be better ( at the moment ) add a stage in the existing code-review pipeline

Reply:
absolutely, you can add a parallel stage to code review pipeline to be faster.
Automated test are very useful on code review, in this way you can fix possible errors before merge your PR (early feedback :) )

Reply:
Ok. I proceed in this way